src/redis/index.ts

```diff
@@ -0,0 +1,19 @@
+import { createClient } from 'redis';
+
+export const client = createClient();
+
+client.on('error', (err) => {
+  console.log('Redis client Error ', err);
+});
+(async function () {
+  await client.connect();
+  console.log('Redis connected successful');
+})();
+
+export const redisCheckName = async (name: string) => {
```

Comment:
Not sure if `redisCheckName` is the best name for this function as it's just a `get` function and it doesn't check for anything.